tests/modules/test_tasks.py

```diff
@@ -40,9 +40,9 @@ def test_get_all_tasks_with_no_params():
     assert isinstance(response.get('total_count'), int)
     assert len(response.get('tasks')) <= 20 and len(response.get('tasks')) >= 1
 
-    task = next(filter(lambda task: task.get('id') == task_id_no_params_test, response.get('tasks')), None)
+    task = next(filter(lambda task: str(task.get('id')) == task_id_no_params_test, response.get('tasks')), None)
     assert isinstance(task, dict)
-    assert task.get('id') == task_id_no_params_test
+    assert str(task.get('id')) == task_id_no_params_test
```

Comment:
Tasks endpoint returns `task_id` as an int whereas the Catalog endpoint returns it as a string. See line 25.

Reply:
Can we also raise this with backend?

Reply:
I've checked and they all endpoints return ints. Reverted the changes.